android/sdk/src/main/java/org/jitsi/meet/sdk/AudioModeModule.java

```diff
@@ -57,7 +61,7 @@
  * Before a call has started and after it has ended the
  * {@code AudioModeModule.DEFAULT} mode should be used.
  */
-class AudioModeModule
+public class AudioModeModule
```

Comment:
@saghul do you have any opinion on making this class public ? I also did similar change to the react instance holder, but I don't like this. Seems there's no other way for Java modules to talk to each other if they're in different packages. I've been scratching my head on how to do it differently, but no good ideas so far.

Reply:
Indeed, I don't like it. I guess the only way is to have a flat namespace, right? I'd be ok with that.

Reply:
I moved everything to the root package

Reply:
Can this be made package private again?